blivet/autopart.py

```diff
@@ -450,16 +445,13 @@ def do_reqpart(storage, requests):
     _schedule_partitions(storage, disks, [], requests=requests)
 
 
-def do_autopart(storage, data, min_luks_entropy=0):
+def do_autopart(storage, data, min_luks_entropy=None):
     """ Perform automatic partitioning.
 
         :param storage: a :class:`~.Blivet` instance
         :type storage: :class:`~.Blivet`
         :param data: kickstart data
         :type data: :class:`pykickstart.BaseHandler`
-        :param min_luks_entropy: minimum entropy in bits required for
-                                 luks format creation
-        :type min_luks_entropy: int
```

Comment:
Thanks, but it should go next to the other parameters not to the block of text below. ;)